<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteVersion</class>
 <widget class="QDialog" name="DeleteVersion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Delete Version</string>
  </property>
  <widget class="QLabel" name="description">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>360</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select a version to delete</string>
   </property>
  </widget>
  <widget class="QListView" name="versionsListView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>360</width>
     <height>180</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="buttonLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>360</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="buttonLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deleteButton">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
